src/main/java/io/ebean/config/dbplatform/DbDefaultValue.java

```diff
@@ -75,14 +95,28 @@ public String convert(String dbDefaultLiteral) {
     if (dbDefaultLiteral == null) {
       return null;
     }
+    if (dbDefaultLiteral.endsWith("}")) {
```

Comment:
I decided to write the timestamps in ODBC compliant form, e.g: { ts 'yyyy-mm-dd hh:mm:ss' } as most platforms can handle them. The values are then converted back for the current platform